api/google_analytics_provider.py

```diff
@@ -50,7 +51,8 @@ def collect_event(self, _db, license_pool, event_type, time, **kwargs):
                     'cd12': "true" if license_pool.open_access else "false",
                 })
         # urlencode doesn't like unicode strings so we convert them to utf8
-        fields = {k: unicode(v).encode('utf8') for k, v in fields.iteritems()}
+        fields = {k: unicodedata.normalize("NFKD", unicode(v)).encode("utf8") for k, v in fields.iteritems()}
```

Comment:
It's not related to the code in this branch. I think the problem was the tests of this method were giving different results on different systems, because `u"pi\u00F1ata"` was being converted to two different UTF-8 strings.

Courteney changed the tests in an earlier commit (324d9bbc77081868cdfa88165390c88318be51bb) which is in master already.